aten/src/ATen/native/Fill.cpp

```diff
@@ -19,6 +19,11 @@ namespace {
 } // namspace
 
 Tensor& fill_out(Tensor& self, Scalar value) {
+  if (self.is_quantized()) {
+    at::Tensor out = at::ones(self.sizes()).to(kFloat) * value;
+    self.copy_(out);
```

Comment:
How are out of range fill values handled?

Reply:
The `copy_` method already handles the logic of quantizing and taking care of the boundaries

Reply:
I will put an inline comment about that in here